src/unit-test.js

```diff
@@ -604,6 +604,7 @@ QUnit.module('Unit | mock-cloud-firestore', (hooks) => {
           name: firebase.firestore.FieldValue.delete(),
           dad: db.collection('users').doc('user_b'),
           modifiedOn: firebase.firestore.FieldValue.serverTimestamp(),
+          activeYears: mockFirebase.firestore.FieldValue.increment(-1),
```

Comment:
I'm referencing `mockFirebase` here because I was not able to update the `firebase` dependency to a version that contains this without running into build issues that I wasn't able to resolve without too much digging. If we can get that working, that would make this more correct.

Reply:
What's the build error you're getting?

Reply:
It says that core-js is not found for importing. As far as I can tell, this has something to do with polyfills and Babel

Reply:
I've investigated it and it's because of some recent babel changes. I've fixed that in the master branch. Could you try rebasing this PR from that?

Reply:
Thank you, I have now rebased and updated the tests to use `firebase.firestore.FieldValue.increment()`, and things seem to be working 👍

Reply:
Thank you @kimroen